Added a special focus point position for VIEW_SITE task

khaykov opened the pull request:
Fixes #12257

This PR moves a position of a focus point for the VIEW_SITE task so it would not be covered by FAB.
![Screenshot_1593054088](https://user-images.githubusercontent.com/728822/85648380-82229000-b655-11ea-883e-33fe8e7a7395.png)

To test:
- Create new site and start Quick Start.
- Navigate to `Customize Your Site` tasks, and select `View Your Site`.
- Find a focus point on new View Site row on MySite screen, and make sure it's not obstructed by FAB.


PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

osullivanchris commented:
@khaykov hey, in #12257 @frosty also captured

> Alternatively, we added extra padding at the very bottom of the view on iOS so that the user can scroll to the end of a list and see ‘below’ the FAB. Could be a route to consider.

also discussed on p2-pbArwn-Gt

Would just like to know your opinion there. When we display FABs, it does seems strange that the last item in a list will always be obscured by the FAB above it on the Z axis. Is there a general way that Android solves this? My suggestion here had been to add extra blank space to the bottom of the list so that you could scroll _beyond_ the list and access the entire last row if necessary. But it doesn't seem a very elegant solution.

![padding](https://user-images.githubusercontent.com/50576199/85704934-affce880-b6d8-11ea-8ecd-65ce1b6e7bfe.png)

renanferrari commented:
>My suggestion here had been to add extra blank space to the bottom of the list so that you could scroll beyond the list and access the entire last row if necessary. But it doesn't seem a very elegant solution.

@osullivanchris AFAIK that's actually considered a good practice on list screens where the FAB doesn't hide on scroll.

khaykov commented:
@renanferrari I fixed the lint issue and also added bottom padding to the screen :)

![Screenshot_1593128835](https://user-images.githubusercontent.com/728822/85806490-0af80500-b704-11ea-9db7-e1659c0a9496.png)

osullivanchris commented:
> I fixed the lint issue and also added bottom padding to the screen :)

Sweet, looks good!
